packages/caldav/README.md

```diff
@@ -0,0 +1,33 @@
+# timeld - CalDAV integration
+
+This packages supports live update of timesheet entries:
+- _from_ a CalDAV end-point
+- _to_ **timeld**.
+
+## installation & configuration
+
+This integration package is included in the gateway.
+
+The integration requires the following configuration:
+- `uri`: calendar URI (https)
```

Comment:
`url`, surely?  Rationale: it has to be reachable, so requires a concrete reference.